cookieninja/vcs.py

```diff
@@ -89,6 +91,11 @@ def clone(
     if repo_type == 'git':
         repo_name = repo_name.split(':')[-1].rsplit('.git')[0]
         repo_dir = os.path.normpath(os.path.join(clone_to_dir, repo_name))
+    if repo_type == 'git-remote-codecommit':
+        # override repo type as it is a git extension
+        repo_type = 'git'
```

Comment:
Why change the type to `git`?

Reply:
Because we actually want to use git to fetch the repository, the difference between this and regular git repositories, is how we parse the name (see lines 90 and 97)